controllers/requests.ts

```diff
@@ -93,6 +97,24 @@ export const updateRequestController = async (req: any, res: any) => {
       body: requestResult,
     };
     await addLog(requestLog.type, requestLog.meta, requestLog.body);
+
+    if (requestResult.state === REQUEST_STATE.APPROVED) {
+      const requestData = await getRequests({ id: requestId });
+      if (requestData) {
+        const { from, until, requestedBy, message } = requestData as OooStatusRequest;
+        const createUserFutureStatusData = {
+          requestId,
+          status: REQUEST_TYPE.OOO,
+          state: statusState.UPCOMING,
+          from,
+          endsOn: until,
+          userId: requestedBy,
+          message,
+        };
+        await createUserFutureStatus(createUserFutureStatusData);
```

Comment:
variable name shouldn't have verb in it.
can we rename it as `userFutureStatusData`

Reply:
Done